repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py

```diff
@@ -426,7 +426,10 @@ def filter_out_pkgs_in_blacklisted_repos(to_install):
         del to_install[pkg]
 
     if blacklisted_pkgs:
-        report_skipped_packages('packages will not be installed due to blacklisted repositories:',
+        # TODO: Describe why they should be installed in the first place (replacement, etc.)
+        report_skipped_packages('packages will not be installed because they are available only in '
+                                'repositories which are either disabled or optional on RHEL8. '
```

Comment:
I'm not an English expert, but wouldn't `that` be more correct than `which`?

Reply:
thx for english lesson, really!